cli/command/spdk_target/add.go

```diff
@@ -0,0 +1,151 @@
+/*
```

Comment:
Why don't we combine those commands into target command?

Reply:
I think it's different from kernel iscsi target, so I define an other spdk_target.

Reply:
> I think it's different from kernel iscsi target, so I define an other spdk_target which is reasonable in my mind.

Actually they all belong to target, we can provide some option to control these differences, like our CurveBS with SPDK.

Reply:
I did it and pushed to this pr.